package.json

```diff
@@ -15,20 +15,25 @@
   "author": "Chapman - Chapman@Apextion.com",
   "dependencies": {
     "apex-util": "^2.0.3",
-    "chai": "^4.1.1",
+    "bufferutil": "^3.0.5",
+    "chai": "^4.1.2",
+    "dependencies": "0.0.1",
     "discord.js": "^11.1.0",
     "dotenv": "^4.0.0",
+    "erlpack": "github:discordapp/erlpack",
     "eslint": "^4.3.1",
     "eslint-config-airbnb": "^15.1.0",
     "eslint-config-airbnb-standard": "^1.4.1",
-    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-import": "^2.13.0",
```

Comment:
Updating ESLint should be a separate branch. Each feature should be in a single unique pull request allowing for a narrowly focused team discussion.